<?xml version="1.0"?>
<launch>
  
  <arg name="serial" default="150534" />
  
  <!-- Ensenso node -->
  <node name="ensenso_driver" pkg="ensenso" type="ensenso_driver" ns="camera" output="screen"> 
    <param name="serial"          type="string" value="$(arg serial)" />
    <param name="camera_frame_id" type="string" value="ensenso_optical_frame" />
    <param name="stream_cloud"    type="bool"   value="true" />
    <param name="stream_images"   type="bool"   value="true" />
    <param name="front_light"     type="bool"   value="false" />
    <param name="projector"       type="bool"   value="true" />
  </node>
  <!-- RViz -->
  <node name="rviz_node" pkg="rviz" type="rviz" args="-d $(find ensenso)/config/viewer.rviz" />
  
</launch>
